src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private String color;
+    private double radius;
+
+    public Circle(double radius, String color) {
+        this.radius = radius;
```

Comment:
```suggestion
    private double radius;

    public Circle(double radius, String color) {
        super(color);
        this.radius = radius;
```

Reply:
I do not agree, which color will be in circle if in figure I do not generate color?